src/components/CustomChart/UploadChartModal.tsx

```diff
@@ -15,6 +16,7 @@ export default function UploadChartModal({ closeUploadPopup }: UploadChartModalT
     const [uploadState, setUploadState] = useState<string>(UPLOAD_STATE.UPLOAD)
     const [errorData, setErrorData] = useState<{ title: string; message: string[] }>({ title: '', message: [] })
     const [loadingData, setLoadingData] = useState(false)
+    const [descriptionLengthError, setDescriptionLengthError] = useState(false)
```

Comment:
Can we change name to hasDescriptionErrorLength or something relevant

Reply:
Done